hw_10/task_2.js

```diff
@@ -0,0 +1,14 @@
+// Напиши функцию countOccurrences, которая принимает массив чисел и возвращает объект с подсчётом каждого числа.
+const numbers = [1, 2, 2, 3, 4, 4, 4, 5];
+
+function countOccurrences(arr) {
+    return arr.reduce((result, element) => {
+        if (result[element]) {
+            result[element]+=1;
+        }else{
+            result[element] = 1;
+        }
+        return result;
+    }, {});
```

Comment:
Не обязательно переписьівать, но как вариант можно и так
```suggestion
    return arr.reduce((result, element) =>
       (result[element] ? result[element] += 1 : result[element] = 1, result), {});
```